hydroshare/urls.py

```diff
@@ -143,7 +143,7 @@
     # one out.
 
     # url("^$", direct_to_template, {"template": "index.html"}, name="home"),
-    url(r"^tests/$", direct_to_template, {"template": "tests.html"}, name="tests"),
+    # QUNIT_TESTS_OFF
```

Comment:
I think you committed the wrong file here, given the commit message.

Reply:
Yes. I undid it.